cds/orgsvc/C#/UploadDownloadFiles/SampleProgram.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Tooling.Connector;
+using System;
+using System.IO;
+
+namespace PowerApps.Samples
+{
+    public partial class SampleProgram
+    {
+        public static CrmServiceClient _service = null;
+        public static string entitySchemaName;
+        private static string columnSchemaName;
+        private const string entityId = "39a9c6f1-20dd-ec11-a7b6-0022482410a9";
```

Comment:
You typically cannot hard code an ID. They are unique to an environment. You will need to do a Retrieve to get the ID unless you created the entity in the program and already have its ID.

The exception is that some templates and security roles have IDs that are fixed across environments and tenants.